CMakeLists.txt

```
@@ -42,9 +43,23 @@ set(C_WARNING_Clang -Wall -Wextra -Werror -Wpedantic
         -Wstrict-prototypes -Wold-style-definition
         -Wno-unused-parameter -Wno-missing-field-initializers -Wno-sign-compare -Wno-language-extension-token)
 
-# Additional compiler-specific flags
-#  -gdwarf-aranges to workaround https://sourceware.org/bugzilla/show_bug.cgi?id=22288 (backtrace line printing with libdw and clang)
 set(C_EXTRA_GNU )
+if (ENABLE_LINKTIME_OPTIMIZATION)
+    include(CheckIPOSupported)
+    check_ipo_supported(RESULT ipo_supported OUTPUT output)
+    if(ipo_supported)
+      message(STATUS "Interprocedural optimization is supported")
```

```suggestion
      message(STATUS "Interprocedural optimization is enabled")
```

me thinks that makes more sense as a status message in cmake output

---

Done !